File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.js
function solution(left, right) {
    var answer = 0;
    for (let j=left; j<= right; j++){
        var count=1;
        for(let i=1; i< j; i++){
            if(j% i===0){
                count++
            }
        }
        if(count %2===0){
            answer+=j
        }else{
            answer-=j
        }
    }
  
    return answer;
}

File: 프로그래머스/1/77884. 약수의 개수와 덧셈/약수의 개수와 덧셈.py
def solution(left, right):
    answer = 0
    while (left <= right):
        ele = []
        for num in range(1,left +1):
            if left % num == 0:
                ele.append(num)
        if len(ele) % 2 == 0:
            answer += num
        else:
            answer -= num
        left = left + 1
    return answer